lib/converter.js

```diff
@@ -49,7 +59,17 @@ function jsonToLang(json, lang, level) {
     if (valueType == "array" || valueType == "object") {
       value = jsonToLang(value, lang, indentationLevel+1);
     } else if (valueType == "string") {
-      value = "'" + value + "'";
+      // Certain languages require double quotes for values
```

Comment:
Honestly we can probably just switch this to always use double quotes. All the other languages so far support using either single or double quotes, so if we just switch to always using double quotes, we can keep the codebase simpler.